index.css

```diff
@@ -31,6 +39,37 @@ header {
   left: 12px;
 }
 
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+
+  /* -webkit-transition: all 0.3s ease-in-out; */
+  transition: all 0.1s ease-in-out;
+  background-color: white;
+}
+
+.hamburger {
+  display: block;
+  z-index: 999 !important;
+  position: absolute;
+  left: 70%;
```

Comment:
Your mobile menu works as intended, however, there is a bit of improvement to be done with regards to the design. Kindly ensure to improve upon the following:
- [ ] The position of your hamburger/close button is wide apart from the right margin, and therefore the position of your hamburger/close button does not match the given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). 
- [ ] Menu items font _(family and weight)_ should match the font found in the chosen [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
- [ ] Background color of the mobile menu should match that found in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
- [ ] Menu items should be vertically centered in the mobile menu.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/196283733-4a3e38c5-ff67-4ed7-8e11-cc0c307f0338.png)


**The Figma design:**
![image](https://user-images.githubusercontent.com/12745474/196283878-eb0a0fea-9c0e-463a-8ed9-6802ce0ae7bd.png)